docs/sql-ref-syntax-qry-explain.md

```diff
@@ -19,4 +19,157 @@ license: |
   limitations under the License.
 ---
 
-**This page is under construction**
+### Description
+
+The `EXPLAIN` statement provides the execution plan for the statement. 
+By default, `EXPLAIN` provides information about the physical plan.
+`EXPLAIN` does not support 'DESCRIBE TABLE' statement.
+
+
+### Syntax
+{% highlight sql %}
+EXPLAIN [EXTENDED | CODEGEN] statement
```

Comment:
We have more modes for explains:
https://github.com/apache/spark/blob/c72f88b0ba20727e831ba9755d9628d0347ee3cb/sql/core/src/main/scala/org/apache/spark/sql/execution/command/commands.scala#L134